Makefile

```diff
@@ -133,7 +138,7 @@ endif
 .PHONY: clean all test lint doc clean_all rcpplint rcppexport roxygen\
 	cython2 cython3 cython cyclean
 
-all: lib/libmxnet.a lib/libmxnet.so $(BIN)
+all: lib/libmxnet.a lib/libmxnet.so $(BIN) cpp-package-example-all
```

Comment:
are cpp-package tests run during CI builds? I think they they probably aren't because they didn't build at all. In the meantime, how will we keep people from "breaking" these compiliables again?